scout/commands/load/research.py

```diff
@@ -122,3 +124,8 @@ def research(case_id, institute, force):
         adapter.case_variants_count(
             case_obj["_id"], case_obj["owner"], "research", force_update_case=True
         )
+
+        if case_missing_all_files:
```

Comment:
Hm, do we? We dont really know that the user requested more than research for the case, and if some category files exist that is fine? The loop is for all types possible, like cancer_research and what not.

I guess the actual file not founds could be considered more exceptional, but we should also warn if there are no research files at all configured for a case.

Reply:
No, I wrote category because I meant if you want to load research and snv research is removed but mei research is present then you had no obvious warnings in the previous version. It was just an example.

Reply:
Right; thats ok. I think it’s good enough like this, though it’s not 100% obvious that it is a problem if one of the files could actually be loaded - maybe that was the only one rerun with research. In that case, with the late warn at least the operator will see it and can investigate, the files that existed will be loaded without delay and most importantly other cases will not be blocked.